.github/workflows/periodic_update.yml

```diff
@@ -0,0 +1,65 @@
+name: Periodic update
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API, or on a schedule.
+on:
+  workflow_dispatch:
+  schedule:
+  # At 10:32 on every day-of-month from 1 through 7 and on Wednesday.
+  # See https://crontab.guru/#32_10_1-7_*_WED
+  - cron: "32 10 1-7 * WED" 
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  update-configure:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
+
+    - name: Install dependencies
+      # Comment out this step if nothing else is needed
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y wget git gawk findutils
+        xargs -a <(awk '! /^ *(#|$)/' ".github/workflows/apt.txt") -r -- \
+            sudo apt-get install -y --no-install-recommends --no-install-suggests
+
+    - name: "Check that autoconf scripts are up-to-date:"
+      run: |
+        rm -f config.guess config.sub
+        wget http://git.savannah.gnu.org/cgit/config.git/plain/config.guess
+        wget http://git.savannah.gnu.org/cgit/config.git/plain/config.sub
+    - run: git diff config.guess config.sub
+    # - run: autoconf
```

Comment:
So autoconf is not changing anything when config.guess and config.sub are changed? I'm just trying to understand if this creates some pending changes or not. If no autoconf is needed, please remove it from here.